src/users/users.controller.ts

```diff
@@ -271,7 +271,7 @@ export class UsersController {
     @Ip() ip: string,
     @Headers('User-Agent') userAgent: string,
   ): Promise<GetUserRespondDto> {
-    let viewerId: number = null;
+    let viewerId: number | undefined = undefined;
```

Comment:
`let viewerId: number | undefined` is enough

Reply:
Fixed.